mlrun/api/crud/client_spec.py

```diff
@@ -22,6 +22,7 @@ def get_client_spec(self):
             artifact_path=config.artifact_path,
             spark_app_image=config.spark_app_image,
             spark_app_image_tag=config.spark_app_image_tag,
+            spark_operator_version=config.spark_operator_version,
```

Comment:
This should go under the `# These have a default value` section (i.e. use `_get_config_value_if_not_default`)